src/d/actor/d_a_player_particle.inc

```diff
@@ -144,11 +299,11 @@ void daPy_swBlur_c::draw() {
     GXSetVtxDesc(GX_VA_TEX0, GX_DIRECT);
     GXSetNumChans(0);
 
-    if (mBlurColorType == 0) {
+    if (mBlurColorType == NORMAL_SLASH_BLUR) {
         // Normal color.
-        static GXColor n_color0 = {0xFF, 0xFF, 0xFF, 0x96};
-        GXSetTevColor(GX_TEVREG0, n_color0);
-    } else if (mBlurColorType == 1) {
+        static GXColor n_color = {0xFF, 0xFF, 0xFF, 0x96};
+        GXSetTevColor(GX_TEVREG0, n_color);
```

Comment:
the correct name of this variable is `n_color0`, I originally had it incorrectly as `n_color` and fixed it a few days ago, looks like it got reverted by a merge conflict